themes/default/content/blog/how-webiny-built-a-serverless-application-framework/index.md

```diff
@@ -210,7 +210,7 @@ These extra resources make it slower to perform deployments from developers' mac
 
 So, with that in mind, we split the API project application’s cloud infrastructure into two stacks - the `dev` and `prod`. And, as you might’ve already guessed, only the `prod` variant will deploy absolutely all necessary cloud infrastructure resources.
 
-What is even more interesting is the fact that this can be achieved with a simple if statement, which we placed in the [`index.ts`](https://github.com/webiny/webiny-js/blob/next/packages/cwp-template-aws/template/api/pulumi/index.ts#L10-L16) entrypoint file:
+What is even more interesting is the fact that this can be achieved with a simple if statement, which we placed in the [`index.ts`](https://github.com/webiny/webiny-js/blob/ea2ea7f956c9f847e0e5e9c48d6b439100991b24/packages/cwp-template-aws/template/ddb-es/api/pulumi/index.ts#L19-L25) entrypoint file:
```

Comment:
do we really want to link to something that's not on their default branch (i.e., next)?

Reply:
no, not ideally.... that code that is being referenced in the post no longer exists on their default branch (or I just wasn't able find it), so this was closest thing I could track down in order to have something to link to.

Reply:
@sean1588 am i missing something? is this not it? https://github.com/webiny/webiny-js/blob/next/packages/cwp-template-aws/template/ddb-es/api/pulumi/index.ts#L19-L25

Reply:
no you are not missing anything at all.... its totally me!  I had done a search and for whatever reason didn’t see anything come up on the main branch. I’ll go ahead and update this link and get this merged up. thank you! 🙂